dummy-extension/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "dummy-extension"
+version = "0.0.1"
+authors = ["John Gallagher <jgallagher@bignerdranch.com>"]
```

Comment:
@gwenn did you write the dummy-extension? If so, shall we change this line to "The rusqlite developers" ?

Reply:
I guess that this file was created before `rusqlite` organisation and not yesterday as suggested by your branch history...
See https://github.com/rusqlite/rusqlite/pull/910#pullrequestreview-605855451

Reply:
Got it - I did start from the phiresky branch but it was a bit of rebasing hell to get all of those commits up to date with master. I ended up squashing everything while I sorted out line by line every single change with respect to master, but I've now replayed my changes on top of the 868 commit history, and indeed it appears I made a mistake during the squashing in which the original authors line was there instead of "The rusqlite developers".